examples/hooks/components/NumericMenu.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { useNumericMenu, UseNumericMenuProps } from 'react-instantsearch-hooks';
+
+import { cx } from '../cx';
+
+export type NumericMenuProps = React.ComponentProps<'div'> &
+  UseNumericMenuProps;
+
+export function NumericMenu(props: NumericMenuProps) {
+  const { hasNoResults, items, refine } = useNumericMenu(props);
+
+  return (
+    <div
+      className={cx(
+        'ais-NumericMenu',
+        hasNoResults && 'ais-NumericMenu--noRefinement',
```

Comment:
should we use canRefine here?

Reply:
It's not returned by the connector unfortunately. Let's maybe add that in IS.js in a minor, and remove `hasNoResults` in the next major?

Reply:
sounds good, I didn't know this widget was missed